apps/src/code-studio/pd/application/teacher/ImplementationPlan.jsx

```diff
@@ -103,9 +103,9 @@ ImplementationPlan.associatedFields = [
 ];
 
 const uniqueRequiredFields = {
-  [PROGRAM_CSD]: ['csdWhichGrades'],
-  [PROGRAM_CSP]: ['cspWhichGrades', 'cspHowOffer'],
-  [PROGRAM_CSA]: ['csaWhichGrades', 'csaHowOffer'],
+  [PROGRAM_CSD]: ['csdWhichGrades', 'enoughCourseHours'],
+  [PROGRAM_CSP]: ['cspWhichGrades', 'cspHowOffer', 'enoughCourseHours'],
+  [PROGRAM_CSA]: ['csaWhichGrades', 'csaHowOffer', 'enoughCourseHours'],
 };
```

Comment:
Could `enoughCourseHours` be in the dynamically required fields? Since it's not specific to CSD/CSP/CSA, I'm not sure it makes as much sense to be in these fields

Reply:
oh good call!